src/LanguageServerProtocol.fs

```diff
@@ -13,6 +13,17 @@ module Server =
   open Ionide.LanguageServerProtocol.JsonUtils
   open Newtonsoft.Json.Linq
 
+  let rec (|HandleableException|_|) (e: exn) =
+    match e with
+    | :? LocalRpcException -> Some ()
+    | :? TaskCanceledException -> Some ()
```

Comment:
if this change doesn't clash with the `revamp` branch, I also recommend  catching `OperationCanceledException`